libnetdata/threads/threads.c

```diff
@@ -134,6 +134,25 @@ static void thread_set_name(NETDATA_THREAD *nt) {
     }
 }
 
+void uv_thread_set_name(uv_thread_t ut, const char* name) {
```

Comment:
I would like the name to be something like `uv_thread_set_name_np` since it's as non-portable as it gets all the way down.

Reply:
ok, should I also rename `thread_set_name` to `thread_set_name_np` ?

Reply:
If you like. But hacking libuv this way is much more serious.

Reply:
Guys, are you sure we need to keep the prefix `uv_` that is used for libuv? This could be a problem case someday libuv decides to bring a function with the same name.

Reply:
This is a hack that may break at any point if a certain libuv version changes the implementation. It's a libuv-specific hack and I think if libuv decides to implement this we should have an `#ifdef` or something with the 2 versions. So, in a way, I think uv has to be part of the name.

Reply:
All right, I will test the PR now.